fuse/new_orleans_pd.py

```diff
@@ -162,21 +150,25 @@ def fuse_events(pprr_ipm, pprr_csd, cprr, uof, award, lprr, sas):
     officer_number_dict = create_officer_number_dict(pprr_ipm)
     cprr = pd.read_csv(deba.data("clean/cprr_new_orleans_pd_1931_2020.csv"))
     actions = pd.read_csv(deba.data("clean/cprr_actions_new_orleans_pd_1931_2020.csv"))
-    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2012_2019.csv"))
+    uof_officers = pd.read_csv(deba.data("clean/uof_officers_new_orleans_pd_2016_2021.csv"))
+    uof_citizens = pd.read_csv(deba.data("clean/uof_citizens_new_orleans_pd_2016_2021.csv"))
+    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2016_2021.csv"))
     post_event = pd.read_csv(deba.data("match/post_event_new_orleans_pd.csv"))
     award = pd.read_csv(deba.data("match/award_new_orleans_pd_2016_2021.csv"))
     lprr = pd.read_csv(deba.data("match/lprr_new_orleans_csc_2000_2016.csv"))
     sas = pd.read_csv(deba.data("match/sas_new_orleans_pd_2017_2021.csv"))
     complaints = fuse_cprr(cprr, actions, officer_number_dict)
-    use_of_force = fuse_use_of_force(uof, officer_number_dict)
-    personnel = fuse_personnel(pprr_ipm, lprr, pprr_csd, sas)
+    personnel = fuse_personnel(pprr_ipm, lprr, pprr_csd, sas, uof_officers)
     events_df = fuse_events(pprr_ipm, pprr_csd, cprr, uof, award, lprr, sas)
     events_df = rearrange_event_columns(pd.concat([post_event, events_df]))
     sas_df = rearrange_stop_and_search_columns(sas)
     lprr_df = rearrange_appeal_hearing_columns(lprr)
+    uof_df = rearrange_use_of_force(uof)
+    citizen_df = rearrange_citizen_columns(uof_citizens)
     complaints.to_csv(deba.data("fuse/com_new_orleans_pd.csv"), index=False)
-    use_of_force.to_csv(deba.data("fuse/uof_new_orleans_pd.csv"), index=False)
+    uof_df.to_csv(deba.data("fuse/uof_new_orleans_pd.csv"), index=False)
     personnel.to_csv(deba.data("fuse/per_new_orleans_pd.csv"), index=False)
     events_df.to_csv(deba.data("fuse/event_new_orleans_pd.csv"), index=False)
     lprr_df.to_csv(deba.data("fuse/app_new_orleans_csc.csv"), index=False)
     sas_df.to_csv(deba.data("fuse/sas_new_orleans_pd.csv"), index=False)
+    citizen_df.to_csv(deba.data("fuse/citizen_new_orleans_pd.csv"))
```

Comment:
I think the file name should have "uof_citizen_" and the prefix.

Reply:
Ok.